bindings/nodejs/src/functions/i2c.cc

```diff
@@ -297,6 +304,7 @@ NAN_METHOD(bind_sol_i2c_write_quick)
     if (!i2c_pending) {
         delete callback;
         hijack_unref();
+        info.GetReturnValue().Set(ReverseLookupConstant("E", abs(errno)));
```

Comment:
```errno``` should never be negative. This is a bug on the native side, confusing a return value (which should be negative) with setting the global ```errno```, which should be positive.